src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -385,6 +385,11 @@ private void AnalyzeNameOfOperation(OperationAnalysisContext operationContext)
                     OnSymbolUsage(memberReference.Member.OriginalDefinition, ValueUsageInfo.ReadWrite);
                     return;
                 }
+                else if (nameofArgument is IPropertyReferenceOperation propertyReference)
```

Comment:
`IPropertyReferenceOperation` is actually an `IMemberReferenceOperation`.
I have no idea why/when we don't get the right IOperation and have to fallback to SemanticModel.